charts/ai-software-templates/chatbot/templates/app-config.yaml

```diff
@@ -9,10 +9,11 @@ metadata:
     app.kubernetes.io/name:  {{ .Release.Name }}-app-config
     app.kubernetes.io/part-of: {{ .Release.Name }}
   name: {{ .Release.Name }}-app-config
-  namespace: {{ .Release.Namespace }}  
```

Comment:
why is this deleted ?  Sorry, I've re-read the PR description a few times now and I'm not equating which of your 1) to 5) points relates to this.

Reply:
Oh I should have added this in the description. The `namespace` is not defined in the ai-lab-app resources. Therefore, I felt it would be better to have the same approach in all of our files.

However I was on the fence between:
* Removing it completeley. In my mind the namespace is defined once we install the helm release.
* Adding it explicitly everywhere, so once we convert the imported files we add a new line with the namespace.

I chosen the first one, only thinking that the namespace is implied. If that's not the case I'm happy to add some new functionality to update my script.

Reply:
Using explicit namespace is better IMO because of the level of nondeterminism with a) whether the namespace context is event set, and b) it is set to where you expect it to be.

I think it should be added back.

Reply:
In all the different spots.

Reply:
It's fine for me. I'll update the script to add it in the imported files instead.

Reply:
@gabemontero I've addressed it in my last two commits:

* [293936e](https://github.com/redhat-ai-dev/ai-lab-helm-charts/pull/31/commits/293936e67d5ed86b400fcc78bf5d528271eba737)
* [a012832](https://github.com/redhat-ai-dev/ai-lab-helm-charts/pull/31/commits/a012832494f3bfb9b8d7086b84975a8c85583934)